Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -167,4 +193,66 @@ private class StaticVersionEdgeConfig : EdgeConfig
     {
         public override string GetLatestVersion() => "83.0.478.37";
     }
+
+    private static async Task<string> TryFindVersionAsync(IDriverConfig driverConfig)
+    {
+        static (string Executable, string[] Arguments) GetWithPowershell(string name) =>
```

Comment:
All this looks very useful for WebDriverManager too. Please add an issue there about adding support for more browsers, referencing this code.

Reply:
Could you link that issue here, please?

Reply:
https://github.com/rosolko/WebDriverManager.Net/pull/186

Reply:
Good news, my WebDriverManager.Net PR just got approved by the owner.
Not merged yet for some reason, but we'd have to wait until the next release to make use of it anyway so I don't mind that.

Reply:
It's merged now 😉

Reply:
It has been released too. I will update our deps soon.